app/controllers/offer/search_label.js

```diff
@@ -0,0 +1,72 @@
+import searchLabelController from "../search_label";
+
+export default searchLabelController.extend({
+  // ----- Arguments -----
+  packageType: null,
+  item: null,
+
+  //--Ajax Methods
+  createItem() {
+    const item = this.get("store").createRecord("item", this.itemParams());
+    this.runTask(
+      item.save().then(itemResponse => {
+        this.set("item", itemResponse);
+        this.addPackage();
+      })
+    );
+  },
+
+  addPackage() {
+    let pkgRecord = this.store.createRecord("package", this.packageParams());
+    this.runTask(
+      pkgRecord.save().then(pkg => {
+        this.transitionToRoute("receive_package", pkg.id, {
+          queryParams: { isUnplannedPackage: true }
+        });
+      })
+    );
+  },
+
+  //--Params helpers
+  itemParams() {
+    const packageType = this.get("packageType");
+    const offer = this.get("model");
```

Comment:
`offer` and `packageType` can be used directly in the returned object, no need to add extra `const` variable for them.